Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/information/InformationFragmentViewModel.kt

```diff
@@ -35,8 +35,15 @@ class InformationFragmentViewModel @AssistedInject constructor(
     }.asLiveData2()
 
     val currentENFVersion = flow {
-        val enfVersion = enfClient.getENFClientVersion()
-            ?.let { "ENF ${context.getString(R.string.information_version).format(it)}" }
+        val enfVersion = enfClient.getENFClientVersion()?.let { v ->
+            val vString = v.toString()
+            val version = "v%s.%s (%s)".format(
+                vString.getOrElse(0) { '0' },
+                vString.getOrElse(1) { '0' },
+                vString
+            )
+            "ENF ${context.getString(R.string.information_version).format(version)}"
```

Comment:
Suggest to change "ENF" to "ENS" to conform to Google's abbreviation of the term, e.g. in https://blog.google/inside-google/company-announcements/update-exposure-notifications/.

Reply:
See https://github.com/corona-warn-app/cwa-app-android/pull/5556#issuecomment-1241773278